rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -1910,4 +1911,136 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
     )
   }
 
+  it should "return an error when `toList` is called with arguments" in {
```

Comment:
we really shouldn't be testing rholang methods from AST level. Maybe in rare cases, where there's no possibility to write given rholang in source form (I'm not even sure we have such cases) - but for sure not for this. Those tests might actually be better done using RhoSpec.

Reply:
RhoSpec is too cumbersome to justify the extra effort at this point. I checked that the methods work in the example files.

Reply:
Then we should make those tests take rholang and produce AST instead of writing the AST by hand. Guess we can handle that in the future though, as it's the issue for most tests in that class.